src/components/CallView/shared/Screen.vue

```diff
@@ -26,9 +26,10 @@
 			class="screen"
 			:class="screenClass" />
 		<VideoBottomBar v-if="isBig"
-			v-bind="$props"
-			:is-big="true"
-			:is-screen="true"
+			:token="token"
+			:shared-data="sharedData"
+			is-big
+			is-screen
```

Comment:
Is it related and `callParticipantModel`, `localMediaModel` are not supposed to be passed?

Reply:
Makes sense